modules/va_facilities/README.yml

```diff
@@ -71,19 +71,37 @@ paths:
         services. Only facilities of type "health" and "benefits" may be filtered by available
         services.
 
+        Alternatively, one can retrieve multiple facilities by id using this endpoint by making a
+        request with a comma-separated list of ids like `?ids=id1,id1`. When requesting facilities
+        in bulk by `id`, the API will return as many ids as it can find matches for and omit any
+        ids where there is no match. It will not return an HTTP error code if it is unable to match
+        a requested `id`. Clients may supply ids up to the limit a their HTTP client enforces for
+        URI path lengths -- usually 2,048 characters.
+
         Results of this operation are paginated. JSON responses include pagination information
         in the standard JSON API "links" and "meta" elements. GeoJSON responses include pagination
         information in the "Link" header.
       operationId: getFacilitiesByLocation
       security:
         - api_key: []
       parameters:
+        - name: ids
+          description: |
+            List of command separated ids of facilities to retrieve in a single request
+          in: query
+          style: form
+          explode: false
+          required: false
+          schema:
+            type: array
+            items:
+              type: string
         - name: bbox[]
```

Comment:
I don't have a strong opinion about this, but we have bbox that is an array but now ids is a delimited list .. for consistency I would see it as an array as well, but that would drop the number of id's that could be requested in the get since you have to repeat the param name `?ids=id1,id2,id3` vs `?ids[]=id1&ids[]=id2&ids[]=id3`

Reply:
I strongly prefer the comma-separated style and think it's consistent with what I've seen in my usage of these kind of bulk request APIs. I'm not sure consistency is a huge deal here just since you can't make a bulk id request and a bounding box request in the same query.